package.json

```diff
@@ -1,12 +1,35 @@
 {
-  "name": "CIAB-Portal",
+  "name": "ciab-portal",
```

Comment:
Why? Mixed Case bugging you too much?

Reply:
Linter appeasement. The "standard" for package names is all lowercase.

(VSCode lints and highlights both PHP and Javascript for me now, on the fly, with the PHP mostly obeying the same rules we've got the GitHub linter using).